blackbox/docs/src/crate/process_test.py

```diff
@@ -215,23 +227,20 @@ def setUp(self):
 
     def test_graceful_stop_none(self):
         """
-        test min_availability: none
+        test `min_availability: none` will stop the node immediately.
+        Causes some shard to become unassigned (no replicas)
```

Comment:
+1 for the expectation description, would be maybe great to describe this at the other tests also